.github/.config.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+const config = require("conventional-changelog-conventionalcommits");
+
+module.exports = config({
+    types: [
+        // Unhide all types except "ci" so that they show up on generated changelog
```

Comment:
I don't get it. We only want to show "feature" and "fix" on the changelog, right? All other is of no interest to users.

Reply:
I would rather show these than no changes tbf